### ISSUE TITLE
Can someone explain what this does using only two words? 

### ISSUE THREAD

**Issue by Paol-M** [RupertBenWiser/Web-Environment-Integrity#125]
This explainer is too long, and I don't have time to read it all.

**Comment by Lana-chan** [comment]
google extortion

**Comment by ator-dev** [comment]
accreditation monopolism

**Comment by zb3** [comment]
more ads

**Comment by JordanMatuschka** [comment]
hurts users

**Comment by McCovican** [comment]
surveillance capitalism

**Comment by dlenski** [comment]
Internet sad

**Comment by gitlimes** [comment]
google overlords

**Comment by coreyreichle** [comment]
Vendor Tax

**Comment by rfkat** [comment]
At its core, it establishes software components called "attesters" that decide whether your device and/or browser is "trustworthy" enough - as defined by the website you are trying to visit. Websites can enforce which "attesters" users must accept, simply by denying everybody access who refuses to bow down to this regime; or who uses attesters that are deemed "inappropriate"; or who is on a platform that does not provide any attesters the website finds "acceptable".

In short: it is specifically designed to destroy the open web by denying you the right to use whatever browser you want to use, on whatever operating system. It is next-level "DRM", introduced by affiliates of a company that already has monopolized the browser market. And the creators of this "proposal" _absolutely know_ what they are attempting here.

**Comment by SteveDesmond-ca** [comment]
Browser monopoly

**Comment by emanuelserpa** [comment]
Google already have the monopoly on multimedia DRM (Widevine), now they will have the monopoly on the web.

They will impose DRM on the web with the excuse of "trusted computing".

Everything for our safety, and for the children, of course.
